index.html

```diff
@@ -22,18 +22,23 @@
 			  color: #fff;
 			}
 			.demo-layout-transparent{
-				background: url('https://getmdl.io/assets/demos/transparent.jpg') center / cover;
+				background: white;

```

Comment:
Tinanggal ko 'yong background image kasi **parang ang OA na tingnan** I just made it white.